packages/api/src/beacon/routes/validator.ts

```diff
@@ -550,24 +550,31 @@ export function getReqSerializers(): ReqSerializers<Api, ReqTypes> {
   );
 
   const produceBlockV3: ReqSerializers<Api, ReqTypes>["produceBlockV3"] = {
-    writeReq: (slot, randaoReveal, graffiti, skipRandaoVerification, opts) => ({
-      params: {slot},
-      query: {
+    writeReq(slot, randaoReveal, graffiti, skipRandaoVerification, opts) {
+      const query: ReqTypes["produceBlockV3"]["query"] = {
         randao_reveal: toHexString(randaoReveal),
         graffiti: toGraffitiHex(graffiti),
         fee_recipient: opts?.feeRecipient,
-        skip_randao_verification: skipRandaoVerification,
```

Comment:
can't we just do this?

```
...(skipRandaoVerification && {skip_randao_verification: ""}),
```

or someting along those lines, also we should add a comment here in the code of why this is done